spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/SparkProcedures.java

```diff
@@ -53,6 +53,7 @@ private static Map<String, Supplier<ProcedureBuilder>> initProcedureBuilders() {
     mapBuilder.put("ancestors_of", AncestorsOfProcedure::builder);
     mapBuilder.put("register_table", RegisterTableProcedure::builder);
     mapBuilder.put("publish_changes", PublishChangesProcedure::builder);
+    mapBuilder.put("generate_changes", GenerateChangesProcedure::builder);
```

Comment:
Are there any alternative names? I am not sure the procedure actually generates changes.
Let's think a bit. It is not bad but I wonder whether we can be a bit more specific.

Reply:
how about `create_change_view` `generate_change_view`?

Reply:
Other options like `register_change_view` `create_changelog_view`